general/solomiia_tymoshchuk/lecture3/_1/MusicalInstrument.java

```diff
@@ -0,0 +1,28 @@
+package general.solomiia_tymoshchuk.lecture3._1;
+
+import javax.sound.midi.Instrument;
+
+public class MusicalInstrument {
```

Comment:
```
Design classes that represent ... musical instruments (at least 3 different one)
```

the goal of the task was to create an hierarchy of musical instrument classes, each with different behaviour represented by music it can play and a unique name